src/Tests/DiagnosticTests/Helpers.cs

```diff
@@ -14,25 +15,25 @@ public sealed partial class UnitTesting
         /// </summary>
         /// <param name="source">string of source code</param>
         /// <returns></returns>
-        private static Compilation CreateCompilation(string source)
-            => CSharpCompilation.Create(
+        private static Compilation CreateCompilation(string source)
+        => CSharpCompilation.Create(
                 assemblyName: "compilation",
                 syntaxTrees: new[] { CSharpSyntaxTree.ParseText(source, new CSharpParseOptions(LanguageVersion.Preview)) },
                 references: new[] { MetadataReference.CreateFromFile(typeof(Binder).GetTypeInfo().Assembly.Location) },
-                options: new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary));
+                options: new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary));
 
         /// <summary>
         /// CreateDriver makes a CSharpGeneratorDriver
         /// </summary>
         /// <param name="compilation"></param>
         /// <param name="generators"></param>
         /// <returns></returns>
-        private static GeneratorDriver CreateDriver(Compilation compilation, params ISourceGenerator[] generators)
+        private static GeneratorDriver CreateDriver(Compilation compilation, AnalyzerConfigOptionsProvider? options, params ISourceGenerator[] generators)
             => CSharpGeneratorDriver.Create(
                 generators: ImmutableArray.Create(generators),
                 additionalTexts: ImmutableArray<AdditionalText>.Empty,
                 parseOptions: (CSharpParseOptions)compilation.SyntaxTrees.First().Options,
-                optionsProvider: null); 
+                optionsProvider: options); 
         // todo: pass the CsWinRTComponent config option here so we don't have to comment out the check in the source generator 
```

Comment:
I think you can drop this comment as your change addresses it.

Reply:
+1